HappyCamps/src/app/login/login.component.ts

```diff
@@ -11,14 +11,22 @@ import { UserStoreService } from '../services/user-store.service';
 })
 export class LoginComponent {
   validateForm!: UntypedFormGroup;
-
+  checked:boolean=true
```

Comment:
if you initialise the value with true it's quite redundant to give it the boolean variable